addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -810,10 +810,9 @@ registerWebsitePreviewTour('website_form_contactus_edition_with_email', {
     edition: true,
 }, () => editContactUs([
     {
-        content: 'Change the Recipient Email',
-        trigger: '[data-field-name="email_to"] input',
-        // TODO: remove && click body
-        run: "edit test@test.test && click body",
+        content: "Change the Recipient Email",
+        trigger: "[data-action-id='addActionField'] input",
```

Comment:
```suggestion
        trigger: "div[data-label='Recipient Email'] input",
```
Why not this instead of using ```addActionField```

Reply:
Sure, we can also use that.